.github/scripts/install_tiledb_source_linux.sh

```diff
@@ -0,0 +1,9 @@
+set -e -x
+git clone https://github.com/TileDB-Inc/TileDB
+cd TileDB
+git checkout 09e665cad9a4e3ed9c28ff3d100a218080fb18e7
```

Comment:
If you want to follow along with the progress on that branch, you could make the first line:

```
git clone -b bd/ch9808-schema-evolution-serialization https://github.com/TileDB-Inc/TileDB.git
```